packages/hdwallet-ledger/src/utils.ts

```diff
@@ -6,12 +6,13 @@ import {
   WrongApp,
   SelectApp,
   ActionCancelled,
+  NavigateToDashboard,
   makeEvent,
 } from '@shapeshiftoss/hdwallet-core'
 import { LedgerTransport } from './transport'
 import { Buffer } from "buffer";
 
-export function handleError (transport: LedgerTransport, result: any, message: string): void {
+export function handleError (transport: LedgerTransport, result: any, message?: string): void {
```

Comment:
Can we have the transport be optional and only emit the error if it exists to avoid duplicating the error handling function?

Reply:
@kaladinlight yes that's a good call. I meant to consolidate this. I will add some logic to throw the error when used outside of an event emitter.